cmake/Modules/GnuradioConfig.cmake.in

```diff
@@ -24,15 +24,23 @@ list(INSERT CMAKE_MODULE_PATH 0 "${CMAKE_CURRENT_LIST_DIR}")
 
 find_dependency(LOG4CPP)
 find_dependency(MPLIB)
-find_dependency(Boost "@Boost_MAJOR_VERSION@.@Boost_MINOR_VERSION@.@Boost_SUBMINOR_VERSION@" COMPONENTS
+
+set(BOOST_REQUIRED_COMPONENTS
     date_time
     program_options
     filesystem
     system
     regex
     thread
-    unit_test_framework
-    )
+)
+
+set(ENABLE_TESTING @ENABLE_TESTING@ CACHE BOOL "Enable testing support")
```

Comment:
I think we want to put this in an `if` clause: `if(NOT ENABLE_TESTING)`.

Rationale: if `ENABLE_TESTING` is defined by the caller, we want to use that setting rather than whatever GR did. I believe that testing in an OOT is independent of GR testing, so we enable it by default here if not set, but otherwise we honor the caller's setting.

What do you think?

Reply:
make sense